Test.Utility/ContainerInterfaceRetriever.cs

```diff
@@ -19,10 +19,15 @@ public static IEnumerable<Type> GetAssemblyInterfaces(IContainer container)
         private static IEnumerable<Type> GetAssemblyTypes(Assembly assembly)
             => assembly
                 .GetTypes()
-                .Where(type => type.IsInterface && !type.IsGenericType && !BelongsToAutofacNamespace(type));
+                .Where(IsValidType);
 
-        private static bool BelongsToAutofacNamespace(Type type)
-            => NamespaceContains(type, nameof(Autofac));
+        private static bool IsValidType(Type type)
+            => type.IsInterface &&
+               !type.IsGenericType &&
+               !type.BelongsToNamespaces(new[] { nameof(Autofac), nameof(System) });
+
+        private static bool BelongsToNamespaces(this Type type, IEnumerable<string> namespaces)
```

Comment:
Slightly inaccurate. How about `BelongsToAnyNamespace`?

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/24/commits/cf32104e6a7ed1adbca3ccdf08097e7c1c0098f4